documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -10,134 +11,125 @@ All development setup is done with a Mac OS
+- create a `.cert` directory in the frontend folder if not already created `mkdir -p ./frontend/.cert`
   - This is where your certificates will be saved.
   - This folder is also excluded from the repository via the .gitignore file.
-- create a key and cert file ```mkcert -key-file ./frontend/.cert/key.pem -cert-file ./frontend/.cert/cert.pem "localhost"```
+- create a key and cert file `mkcert -key-file ./frontend/.cert/key.pem -cert-file ./frontend/.cert/cert.pem "localhost"`
 
+## Set up and Deploy Database
 
-## Setup and Deploy Database
 The database is a postgres database, which needs to be deployed via docker compose.
+
 - Add a `./backend/.env` by either creating a new file or using [sample.env](https://github.com/bcgov/gdx-agreements-tracker/blob/development/backend/sample.env) as the template
-- Update/add all database [environment variables]
-```sh 
+- Update/add all database environment variables
+
+```sh
 #/backend/.env
 ...
+DATABASE_AUTO_DEPLOY=1
+POSTGRES_DB=gat_db
+POSTGRES_HOST=0.0.0.0
+POSTGRES_PASSWORD=postgres
 POSTGRES_PORT=15432
-POSTGRES_HOST=localhost
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD=postgres
-POSTGRES_DATABASE=gat_db
-DATABASE_AUTO_DEPLOY=1
 ...
 ```
-- Start the database by running the docker compose command ```cd ./backend && docker compose up --build```
-  - if this fails you can manually build the image ```docker build -t backend-db ./openshift/templates/images/postgres/``` and then run ```cd ./backend && docker compose up```
 
+- Start the database by running the docker compose command:
 
-## Backend Setup and Deploy
-- Create/update an .env for the backend
+```bash
+cd ./backend
+docker volume create gdx-agreements-tracker_database # set up a volume to hold the database
+docker compose up --build
+```
 
-```sh
-# Json web token uri
-JWKSURI=https://example.com/auth/realms/aaoozhcp/protocol/openid-connect/certs
-# Node env
+- If the --build step fails you can manually build the image:
+
+```bash
+ docker build -t backend-db ./openshift/templates/images/postgres/
+ docker compose up
+```
+
+## Set up Back-End API
+
+- Use the [Set Up and Deploy Database](#set-up-and-deploy-database) instructions above to create your .env file, or Create`./backend/.env` _(see example below)_ using [sample.env](https://github.com/bcgov/gdx-agreements-tracker/blob/development/backend/sample.env) as a template:
+
+```bash
+# Refer here for configs: https://console.apps.silver.devops.gov.bc.ca/k8s/ns/acd38d-dev/configmaps/0-gdx-agreements-tracker-api-env-config/yaml
+# Refer here for secrets: https://console.apps.silver.devops.gov.bc.ca/k8s/ns/acd38d-dev/secrets/pmo-secrets-7f57hmd56g/yaml
+
+# JSON Web Key Set Universal Resource Identifier
+JWKSURI=
+# Node environment
 NODE_ENV=development
-# Database
-POSTGRES_PORT=
-POSTGRES_HOST=
-POSTGRES_USER=
-POSTGRES_PASSWORD=
-POSTGRES_DATABASE=
+# Database Config
 DATABASE_AUTO_DEPLOY=1
+POSTGRES_DATABASE=gat_db
+POSTGRES_HOST=localhost
```

Comment:
not consistent with your previous setup of database host ie localhost != 0.0.0.0